python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -52,6 +64,645 @@
 DEFAULT_FILE_TYPE = "csv"
 
 
+@PublicAPI(stability="beta")
+class NewExperimentAnalysis:
```

Comment:
Most of this class is the same, except the init logic is simplified, we now read the json/csv files from the filesystem directly (instead of downloading), and modified the internal data structures a bit.